mp/src/game/shared/neo/neo_version.cpp

```diff
@@ -3,12 +3,11 @@
 #include "convar.h"
 #include "dbg.h"
 
-namespace {
-void neoVersionCallback()
+void NeoVersionPrint()
 {
 #if defined(GAME_DLL)
 	static constexpr char HEADER[] = "neo_version (Server's build info):";
-#else defined(CLIENT_DLL)
+#elif defined(CLIENT_DLL)
 	static constexpr char HEADER[] = "neo_version (Client's build info):";
 #endif
 	Msg("%s\n"
```

Comment:
```suggestion
	Msg("%s\n"
```
`Msg(static_cast<tchar>("%s\n"`?

Reply:
No